main-app/app/scripts/directives/directive.js

```diff
@@ -1,23 +1,19 @@
 (function () {
     'use strict';
-    angular.module('tombola.snapCardGame.directives')
+    angular.module('Tombola.SnapCardGame.Directives')
 
         .directive('cardSpace', function () {
             return {
                 restrict: 'E',
                 replace: true,
-                template: '<div class="snapCard showImg{{gameModel.getResultSprite()}}"></div>'
-
+                template: '<div class="snapCard showImg{{gameModel.getResultSprite}}"></div>'
             };
         })
-
     .directive('cardSpace1', function () {
```

Comment:
This should be in its own file, if it is needed at all.
Don't like the naming, what does cardSpace1 actually mean? This is a bad code-smell (the smell is bad rather than the code), I suspect either the architecture needs looking at or getResultSprite function needs a param.